DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -158,18 +166,38 @@ void Executor::operator()(const Model::DispatchCommand& command)
             }
 
             PIXBeginEvent(m_device->GetCommandQueue(), PIX_COLOR(255, 255, 0), "Dispatch '%s'", command.dispatchableName.c_str());
-            
-            dispatchable->Dispatch(command);
-            
+
+            uint32_t timestampIndex = (iteration * 2) % timestampCount;
+
             if (dispatchableUsesDeviceCommandList)
             {
+                m_device->GetCommandList()->EndQuery(m_device->GetTimestampHeap(), D3D12_QUERY_TYPE_TIMESTAMP, timestampIndex);
+                dispatchable->Dispatch(command);
+                m_device->GetCommandList()->EndQuery(m_device->GetTimestampHeap(), D3D12_QUERY_TYPE_TIMESTAMP, timestampIndex + 1);
                 m_device->DispatchAndWait();
             }
+            else
+            {
+#ifdef WIN32
```

Comment:
I added this WIN32 check around the Onnx specific code

Reply:
Ah, try this instead: `#ifndef ONNXRUNTIME_NONE`.